docs/reference/indices/put-component-template.asciidoc

```diff
@@ -0,0 +1,213 @@
+[[indices-component-template]]
+=== Put component template API
+++++
+<titleabbrev>Put component template</titleabbrev>
+++++
+
+Creates or updates a component template. 
+Component templates are building blocks for constructing <<indices-templates,index templates>>  
+that specify index <<mapping,mappings>>, <<index-modules-settings,settings>>, 
+and <<indices-aliases,aliases>>. 
+
+[source,console]
+--------------------------------------------------
+PUT _component_template/template_1
+{
+  "template": {
+    "settings": {
+      "number_of_shards": 1
+    },
+    "mappings": {
+      "_source": {
+        "enabled": false
+      },
+      "properties": {
+        "host_name": {
+          "type": "keyword"
+        },
+        "created_at": {
+          "type": "date",
+          "format": "EEE MMM dd HH:mm:ss Z yyyy"
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// TESTSETUP
+
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+DELETE _component_template/template_*
+--------------------------------------------------
+// TEARDOWN
+
+//////////////////////////
+
+[[put-component-template-api-request]]
+==== {api-request-title}
+
+`PUT /_component_template/<component-template>`
+
+
+[[put-component-template-api-desc]]
+==== {api-description-title}
+
+An index template can be composed of multiple component templates. 
+To use a component template, you specify it the index template's `composed_of` list.
+You cannot use component templates independently, 
+they are only applied to new data streams and indices if they are part of a matching index template. 
```

Comment:
This is a comma splice.

> You cannot use component templates independently

`Independently` seems a little unclear here. In some ways, modularity = independence.

It may be more clear to say you cannot apply or use component templates with an index template.